internal/server/server.go

```diff
@@ -41,12 +42,8 @@ func Run() error {
 		done := make(chan bool)
 		go func() {
 			for d := range amqpMsgs {
-				amqpLogHandler.HandleLog(d)
```

Comment:
nit: Please explicitly add a comment indicating that this is a deprecated handler.